source/installation/deploy-minio-single-node-single-drive.rst

```diff
@@ -11,26 +11,42 @@ Deploy MinIO as Single-Node Single-Drive
 The procedures on this page cover deploying MinIO in a Single-Node Single-Drive (SNSD) configuration for early development and evaluation.
 This mode was previously called :guilabel:`Standalone Mode` or 'filesystem' mode.
 
-For extended development or production environments, deploy MinIO
-in :guilabel:`Distributed Mode`. See :ref:`deploy-minio-distributed` for more
-information.
+Starting with :minio-release:`RELEASE.2022-06-02T02-11-04Z`, MinIO implements a zero-parity erasure coded backend for single-node single-drive deployments.
+This feature allows access to :ref:`erasure coding dependent features <minio-erasure-coding>` without the requirement of multiple drives.
+
+MinIO only starts in |SNSD| mode if the storage volume or path is empty *or* only contain files generated by a previous |SNSD| deployment.
+See :ref:`minio-snsd-pre-existing-data <minio-snsd-pre-existing-data>` for more complete documentation on MinIO startup behavior in |SNSD| mode.
+
+For extended development or production environments, deploy MinIO in :guilabel:`Distributed Mode`. See :ref:`deploy-minio-distributed` for more information.
+
+.. _minio-snsd-pre-existing-data:
 
 Pre-Existing Data
 -----------------
 
-Starting with :minio-release:`RELEASE.2022-06-02T02-11-04Z`, MinIO implements a zero-parity erasure coded backend for single-node single-drive deployments.
-This feature allows access to :ref:`erasure coding dependent features <minio-erasure-coding>` without the requirement of multiple drives.
-The specified data volume or folder *must* be empty *or* only contain files generated by a previous |SNSD| deployment.
+MinIO startup behavior depends on the the contents of the specified storage volume or path.
+The server checks for both MinIO-internal backend data and the structure the structure of existing folders and files.
+The following table lists the possible storage volume states and MinIO behavior:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 40 60
+
+   * - Storage Volume State
+     - Behavior
 
-Use an earlier version of MinIO to retain the legacy filesystem-access behavior.
+   * - Empty with no files or MinIO backend data.
+       
+     - MinIO starts in |SNSD| mode and creates the zero-parity backend.
 
-.. admonition:: MinIO Avoids Root Disks
-   :class: important
+   * - Existing |SNSD| zero-parity objects and MinIO backend data.
+     - MinIO resumes in |SNSD| mode.
 
-   By default MinIO returns an error if the specified path is part of the root or OS drive.
+   * - Existing filesystem folders, files, and MinIO backend data
```

Comment:
All other cells end with a period.

Reply:
I think I'd rather the whole table not have periods. The left column does not have any complete sentences.